.github/stale.yaml

```
@@ -0,0 +1,24 @@
+# Configuration for stalebot.
+# This bot automatically marks inactive issues as stale, and eventually closes them.
+
+# Limit this to only issues (not PRs).
+only: issues
+# Number of days of inactivity before an issue becomes stale.
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed.
+daysUntilClose: 7
+# Issues with these labels will never be considered stale.
+exemptLabels:
+  - triage-needed
+  - never-stale
+# Label to use when marking an issue as stale.
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable.
+markComment: >
+  This issue has been automatically marked as stale because it has not had
```

```suggestion
  This issue has been automatically marked as stale because it hasn't had
```
per Acrolinx/ brand voice